app/screens/settings/Settings.tsx

```diff
@@ -472,6 +473,12 @@ class Settings extends Component<Props, State> {
   toggleSignMessageModal = ({ index }: { index: number }) => {
     this.setState({ signMessageModalAccountIndex: index });
   };
+
+  toggleTheme() {
+    const { switchTheme, isDarkMode } = this.props;
```

Comment:
In the future, we'd have a third skin so it will make sense.
But until then I think that we have to keep such logic in a single place: reducer.
See https://github.com/spacemeshos/smapp/pull/807/files#r707151940

Reply:
Fixed